sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosScripts.java

```diff
@@ -236,42 +230,6 @@ public CosmosTrigger getTrigger(String id) {
                                  asyncScripts.getTrigger(id));
     }
 
-    /**
-     * Map stored procedure response and block cosmos sync stored procedure response.
-     *
-     * @param storedProcedureResponseMono the stored procedure response mono
-     * @return the cosmos sync stored procedure response
-     */
-    CosmosStoredProcedureResponse mapStoredProcedureResponseAndBlock(
```

Comment:
why isn't this needed anymore? in the absence of this method does the code throw the valid type of exception (if any failure)?

Reply:
It will still throw the valid exception type. That was just wrapping the exception to CosmosException, which is what originally gets thrown by async APIs.

Reply:
do we have tests to validate that?

Reply:
I reverted the change in the latest iteration to make sure we are consistent with exceptions across the SDK. Thanks for bringing this up. 

Will also add tests to check the exception type.

Reply:
In another follow up PR.